specification.md

```diff
@@ -3266,6 +3265,50 @@ Note that using metadata for cases such as above heavily reduces the portability
 Function definitions themselves do not define data input parameters. Parameters can be
 defined via the `parameters` property in [function definitions](#FunctionRef-Definition) inside [actions](#Action-Definition).
 
+###### AuthRef Definition
+
+| Parameter | Description | Type | Required |
+| --- | --- | --- | --- |
+| resource | References an auth definition to be used to access the resource defined in the operation parameter | string | yes |
+| invocation | References an auth definition to be used to invoke the operation | string | no |
+
+The `authRef` property references a name of a defined workflow [auth definition](#Auth-Definition).
+It can be a string, in which case the referenced [auth definition](#Auth-Definition) is used solely for the function's invocation, or an object, in which case it is possible to specify an [auth definition](#Auth-Definition) to use for the function's resource retrieval (as defined by the `operation` property) and another for its invocation.
+
+Example of a function definition configured to use an [auth definition](#Auth-Definition) called "My Basic Auth" upon invocation:
+
+```yaml
+functions:
+- name: SecuredFunctionInvocation
+  operation: https://test.com/swagger.json#HelloWorld
+  authRef: My Basic Auth
+```
+
+Example of a function definition configured to use an [auth definition](#Auth-Definition) called "My Basic Auth" to retrieve the resource defined by the `operation` property, and an [auth definition](#Auth-Definition) called "My OIDC Auth" upon invocation:
+
+```yaml
+functions:
+- name: SecuredFunctionInvocation
+  operation: https://test.com/swagger.json#HelloWorld
+  authRef:
+    resource: My Basic Auth
+    invocation: My OIDC Auth
+```
+
+Note that if multiple functions share the same `operation` value, and if one of them defines an [auth definition](#Auth-Definition) for resource access, then it should always be used to access said resource.
```

Comment:
I think here we should explicitly say that it's only the file part of the string, before the operation id. Because "operation value" might lead to misunderstandings.